docs/en/action-menu-items/custom-browser-protocols.md

```diff
@@ -174,10 +174,86 @@ on open location this_URL
 end open location 
 ```
 
-> **Tip:** To ensure you are running Python from a specific shell, such as tcsh, you can change the do shell script for something like the following:  
-> 
->     do shell script "tcsh -c \"sgTriggerScript.py '" & this_URL & "'\""
-In the Script Editor, save your short script as an “Application Bundle”.
+{% include info title="Tip" content="To ensure you are running Python from a specific shell, such as tcsh, you can change the do shell script for something like the following:  
+    do shell script "tcsh -c \"sgTriggerScript.py '" & this_URL & "'\""
```

Comment:
Can you apply code format "`" to the tcsh command?

Reply:
Waiting on the latest preview to confirm things look good.